### PR TITLE
Progress dialog: Fix remaining time

### PR DESCRIPTION
**Pull request by Megamouse**
It was showing the total time.
Also removes the 00m if the time is less than 60 seconds.

(Ignore the first commit, it's a rebase)

### CODE REVIEW

File: rpcs3/Emu/system_progress.cpp [hunk 1]
```diff
@@ -252,6 +253,10 @@ void progress_dialog_server::operator()()
 						{
 							fmt::append(progr, " (%um remaining)", minutes);
 						}
+						else if (minutes == 0)
+						{
+							fmt::append(progr, " (%02us remaining)", seconds);
```
Comment (elad335) [inline rpcs3/Emu/system_progress.cpp:258]: But the minutes being 0 helps the user notice that it is seconds.

Comment (Megamouse) [reply rpcs3/Emu/system_progress.cpp:258]: Did you just assume the users can't read?

Comment (Megamouse) [reply rpcs3/Emu/system_progress.cpp:258]: In seriousness, this just looks weird and does not provide any meaningful information.
Any designer will tell you to drop it

Comment (elad335) [reply rpcs3/Emu/system_progress.cpp:258]: I assume that is a single letter difference is not very noticeable and that the MM:SS format is very recognizable.

**Pull request merged**